glide-core/src/client/value_conversion.rs

```diff
@@ -180,6 +181,24 @@ pub(crate) fn convert_to_expected_type(
             )
                 .into()),
         },
+        ExpectedReturnType::ArrayOfDoubles => match value {
```

Comment:
maybe we should have a function called convert to array of, since its identical to ArrayOfBools

Reply:
Good idea, I added a convert_array_elements function